src/allotropy/parsers/appbio_quantstudio_designandanalysis/structure/generic/structure.py

```diff
@@ -147,9 +157,21 @@ def create(
             f"Unable to find well position for Well '{identifier}'.",
         ]
 
-        amp_data = contents.get_non_empty_sheet("Amplification Data")
-        melt_curve_data = contents.get_non_empty_sheet_or_none("Melt Curve Raw")
+        amp_data = cls.get_amplification_data_sheet(contents)
+        amplification_data = (
+            AmplificationData.create(amp_data, identifier, target_dna_description)
+            if amp_data is not None
+            else None
+        )
 
+        melt_data = contents.get_non_empty_sheet_or_none("Melt Curve Raw")
```

Comment:
General comment but not specific to this pr, I wonder if a constant text string like this should in this case be a property of `MeltCurveData` or the corresponding creator class

Reply:
as in a class level prop

Reply:
it is a good idea.

I'll implement for all sheets names in a separated PR

Reply:
💯